evm/its.js

```diff
@@ -362,12 +375,14 @@ async function processCommand(config, chain, options) {
 
             const tokenIdBytes32 = hexZeroPad(tokenId.startsWith('0x') ? tokenId : '0x' + tokenId, 32);
 
-            const tx = await interchainTokenService.callContractWithInterchainToken(
+            const metadata = concat(['0x00000000', data]);
+
+            const tx = await interchainTokenService.interchainTransfer(
```

Comment:
i'd remove this action altogether. this prefixing can be done for `interchainTransfer` instead for simpler UX

Reply:
removed